resolwe/flow/executors/__init__.py

```diff
@@ -4,379 +4,4 @@
 Flow Executors
 ==============
 
-.. autoclass:: resolwe.flow.executors.BaseFlowExecutor
```

Comment:
Ughhh... such moves & changes are very hard to review.

Please, do the moves in a preparatory commit ([like this one](https://github.com/genialis/resolwe/commit/8f3132ed24cbbd61dd336004c3dead53385eabf9)) and then squash all other changes in a combined commit ([like this one](https://github.com/genialis/resolwe/commit/4624f583d7ff40dbdea2dd85488e144a4705ee5d)).

The combined changes will still be hard to review ([an example](https://github.com/genialis/resolwe/compare/master...tjanez:jan-separate-exec2)) but a reviewer could select the second (combined) commit that would be nicer to review.

I've done this already so just let me know if I can push the changes to your branch.

Reply:
This has been done.